contracts/tests/TestController.sol

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-License-Identifier: UNLICENSED
+ */
+pragma solidity 0.6.10;
+
+// import "../packages/oz/upgradeability/VersionedInitializable.sol";
+import "../interfaces/OtokenInterface.sol";
+
+/**
+ * @author Opyn Team
+ * @notice Upgradeable Controller testing contract
+ */
+contract TestController {
```

Comment:
hmm after looking at how a bunch of other teams do it, maybe we want to call this also a MockController contract? A TestController Id imagine inherits the controller and tests it?